src/indep_adms/agent/adm_amp_agent_impl.c

```diff
@@ -1806,11 +1807,12 @@ tnv_t *amp_agent_ctrl_add_tbr(eid_t *def_mgr, tnvc_t *parms, int8_t *status)
 	rule_t *tbr = NULL;
 
 	ari_t *id = adm_get_parm_obj(parms, 0, AMP_TYPE_ARI);
-	OS_time_t start = amp_tv_to_ctime(adm_get_parm_tv(parms, 1, &success), NULL);
```

Comment:
Should the start time actually be an absolute time based on a non-null second argument?

Reply:
The actual rule handling is conditioned on the start time being relative or absolute, so no special handling is needed in the CTRL conversion here.